protos/camera/camera.proto

```diff
@@ -311,6 +320,18 @@ message VideoStreamInfo {
     Status status = 2; // Current status of video streaming
 }
 
+// Type for video stream status. Difference to VideoStreamSettings: Status gets sent whenever there is a change, e.g. fov of the camera. Settings get queried usually once during startup.
+message VideoStreamStatus{
+    bool is_running = 1; // Video stream is running / not running
```

Comment:
This is e.g. a duplicate with `Status` (`STATUS_NOT_RUNNING` and `STATUS_IN_PROGRESS`).